### DISCUSSION TITLE
Does the Pi Zero 2 get just 1 Tone generator?

### DISCUSSION THREAD

**Discussion by Doug2233** [probonopd/MiniDexed#319] [Q&A] [answered]
I can understand the Zero being limited to a single tone generator but the Zero 2 is a much more capable 'Zero' on par with many of the older models.  The github intro paragraph is pretty explicit about the Zero, I'm hoping for 8 TGs on the Zero 2.   
Thanks,
Doug

**Comment by Banana71** [comment, marked as answer]
As I understand it, the Zero 2 can play all 8 TGs.
I would love to check that, but unfortunately it's very difficult to get a Zero 2 at the moment.

**Reply by probonopd** [reply]
> As I understand it, the Zero 2 can play all 8 TGs.

Correct.

It works, I have tested it :+1:

**Reply by Doug2233** [reply]
Thanks again to everyone who is contributing to this project.  So much fun digging into last century sounds with my standard P4 setup.   MiniDexed just keeps on giving. Looking forward now to building a Zero 2 ‘loaner’ version for my musician friends.

**Reply by probonopd** [reply]
Thanks for your kind remarks, keep spreading the word!
